ajax/libs/jquery-outside-events/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "jquery-outside-events",
+  "filename": "jquery.ba-outside-events.min.js",
+  "version": "1.1",
+  "homepage": "https://github.com/cowboy/jquery-outside-events",
+  "keywords": [
+    "jQuery",
+    "clickoutside"
```

Comment:
Is this `clickoutside` made by yourself?

There are 3 more *-outside plugin: 
https://github.com/cowboy/jquery-outside-events/blob/v1.1/jquery.ba-outside-events.js#L30-L33

I will choose to use outside only.